### PR TITLE
Add additional States Daily pub sheet CSV columns

### PR DESCRIPTION
**Pull request by zachlipton**
- Add support for columns that are always blank
- Add test for CSV generator

This got slightly more involved than expected because we need a totalTestResults column that's always blank to match the existing public sheet.

### CODE REVIEW

File: tests/app/csv_test.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"""
+Tests for CSV generation code (CSV endpoints are tested in ``public_test.py``
```
Comment (julia326) [inline tests/app/csv_test.py:2]: actually - maybe also move the CSV endpoint tests here too, since this is where the CSV endpoints are?

**Review by julia326 — COMMENTED** [review]
pretty much LGTM just a couple minor things

File: tests/app/csv_test.py (hunk 1 above)
Comment (zachlipton) [reply tests/app/csv_test.py:2]: Sure. Everything's in csv_test.py now.

File: app/api/csv.py [hunk 2]
```diff
@@ -11,7 +11,8 @@
 from app.api import api
 from app.api.common import us_daily_query, states_daily_query, State
 
-CSVColumn = collections.namedtuple('Column', 'label model_column')
+CSVColumn = collections.namedtuple('Column', 'label model_column blank')
+CSVColumn.__new__.__defaults__ = (False, )
```
[the remark this answers is not in the document]
Comment (zachlipton) [reply app/api/csv.py:16]: Good idea. It was a bit obscure because the defaults works from right to left. Added a comment.

**Review by julia326 — APPROVED** [review]

**Pull request merged**